src/utils/is-valid-term.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Reports whether the input looks like an HPO term object.
+ *
+ * NOTE: This function tests for HPO terms returned from the search service, which merges
+ * plain language symptom descriptions into the HPO term objects.
+ *
+ * @param {Any} input
+ * @return {Boolean} true if the input is an object with `id`, `label`, and `symptomText`
+ * properties, and the ID is a string matching the pattern of an HPO term. False otherwise.
+ */
+export default function isValidTerm(input) {
+  return !!(input && input.id && input.label && input.symptomText &&
+      /HP:\d{7}/.test(input.id));
+}
```

Comment:
I find the !! convention for type coercion is a bit hard to read. Might make a good helper method; something like `ensure`.

Reply:
How would you feel about using `Boolean` instead?

```javascript
  return Boolean(input && input.id && input.label && input.symptomText &&
      /HP:\d{7}/.test(input.id));
```

Reply:
Works for me.